ext/drwrap/drwrap.c

```diff
@@ -2401,6 +2405,16 @@ drwrap_is_post_wrap(app_pc pc)
     return res;
 }
 
+DR_EXPORT
+bool
+drwrap_get_stats(INOUT drwrap_stats_t *stats)
+{
+    if (stats == NULL || stats->size != sizeof(*stats))
+        return false;
+    stats->flush_count = drwrap_stats.flush_count;
```

Comment:
Ok to read without some sort of acquire load?

Reply:
Actually thank you for raising this: for 32-bit ARM this does need something.  I will put in an atomic for all platforms for clarity.